src/hooks/useColumns.tsx

```diff
@@ -82,6 +82,28 @@ function warningFixed(flattenColumns: { fixed?: FixedType }[]) {
   }
 }
 
+function revertForRtl<RecordType>(columns: ColumnsType<RecordType>): ColumnsType<RecordType> {
+  return columns.map(value => {
```

Comment:
`value`  => `column`

For follow `'children' in column`, it need nest column to test that. You can ref this test: https://github.com/react-component/table/blob/master/tests/GroupingColumns.spec.js

Reply:
> `value` => `column`
> 
> For follow `'children' in column`, it need nest column to test that. You can ref this test: https://github.com/react-component/table/blob/master/tests/GroupingColumns.spec.js

changed, and calling `revertForRtl` after flattening columns, so it don't even need children.

https://github.com/react-component/table/blob/da83eada45ea20d2ca7a48ae1b8ed2247c80e77a/src/hooks/useColumns.tsx#L193-L197